src/apis/reviews.js

```diff
@@ -1,14 +1,21 @@
 import axiosBaseUrl from "./connection";
 
-export const getReviews = () => {
-  return axiosBaseUrl
-    .get("/reviews")
-    .then((response) => {
-      return response.data.reviews;
-    })
-    .catch((err) => {
-      console.log(err);
-    });
+export const getReviews = async (searchParams) => {
+  try {
+    let response = [];
+    if (searchParams === "") {
```

Comment:
this should work without the need of an if condition (searchParams should be ignored by axios if its value is undefined, give it a go!)

Reply:
This is how I have done first but I was getting errors when there is no value passed, so I had to add the if condition

Reply:
I will try it again and tell you what error I am getting